source/Testing/Mods/TestModControlParams/TestModControlParams.csproj

```diff
@@ -34,6 +34,14 @@
     <None Remove="Reloaded.Checks.targets" />
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
There's also some leftover automatic additions from your IDE. This should be redundant as OutputPath is set on top of this file.